applications/ironside/se/update/README.rst

```diff
@@ -0,0 +1,56 @@
+.. _ironside_se_update_application:
+
+IRONside SE Update
+##################
+
+The *IRONside SE Update* application updates the IRONside SE firmware on a SoC that already has IRONside SE installed.
+It can update both the main image and the recovery image of IRONside SE using the IRONside SE firmware release ZIP file.
+
+Update procedure
+****************
+
+The application invokes the IRONside SE update service and passes the parameters that correspond to the location of the HEX file of the IRONside SE firmware update.
+   
+The application prints the return value of the service call and outputs information from the update HEX file.
+
+After the service call completes, the IRONside SE firmware updates the internal state of the device.
+The firmware installs the update during the next device boot.
```

Comment:
I'd add that the update process needs a short time period and can be verified with reading back the version via the boot report